pftools/python/parflow/tools/io.py

```diff
@@ -55,7 +55,8 @@ def _decorator(func):
     from yaml import Dumper as YAMLDumper
 
 
-def read_pfb(file: str, keys: dict=None, mode: str='full', z_first: bool=True):
+def read_pfb(file: str, keys: dict=None, mode: str='full', z_first: bool=True,
+             read_sg_info: bool=False):
```

Comment:
The default behavior should be correctness so default for read_sg_info should be True.   Correctness first and optimizations as options, especially in cases if it fails to work correctly.